app/controllers/orders_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @order = Order.find(params[:id])
+    @order_items = @order.order_items
```

Comment:
oh! yeah, I originally had something, to this effect, in here - interesting that was part of the fix - I'm sorry